cmd/generate.go

```diff
@@ -187,7 +188,11 @@ func (gc *generateCmd) run() error {
 		log.Fatalf("error in SetPropertiesDefaults template %s: %s", gc.apimodelPath, err.Error())
 		os.Exit(1)
 	}
-	template, parameters, err := templateGenerator.GenerateTemplate(gc.containerService, engine.DefaultGeneratorCode, BuildTag)
+	bts, _ := json.Marshal(gc.containerService)
```

Comment:
I suppose that's for debugging purposes while developing, if it's there to stay, more explicit variable name

Reply:
reviving this comment